app.py

```diff
@@ -28,18 +24,72 @@ def inject_func():
                 len=len)
 
 
-## initiate ###################
+###### before and after each request ########
+@app.before_request
+def open_conn():
+    global connections
+    conn = connections.getconn()
+    cursor = conn.cursor(cursor_factory = DictCursor)
+    g.cur = cursor
+    g.conn = conn
+    g.error = False
```

Comment:
What's this?

Reply:
shit! It's the leftover of a dropped idea!